packages/bot-web-ui/src/assets/sass/quick-strategy.scss

```diff
@@ -35,6 +35,7 @@
     &__form {
         @include mobile {
             padding: 0 1.6rem;
+            height: 100%;
```

Comment:
IIRC we had set a `min-height` value to maintain one height for the quick strategy modal, there might be a difference in height for the three tabs.

Reply:
That's not needed because its been taken care of in `ThemedScrollbar`'s `height` prop now.